app/scripts/controllers/SysinfoController.coffee

```diff
@@ -103,5 +105,16 @@ angular.module('neo4jApp.controllers')
       $scope.autoRefresh = !$scope.autoRefresh
       refreshLater()
 
+    $scope.isCurrentConnection = (addresses) ->
+      if Settings.useBolt
+        boltHost = Settings.boltHost || $rootScope.boltHost
+        boltAddressForMember = Utils.getServerAddressByProtocol('bolt', addresses)[0]
+        return boltAddressForMember && boltAddressForMember.includes(boltHost)
```

Comment:
And now that the port has been registered, it's unlikely to change... http://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml?search=7687